TempleDAO/temple
# Add OTC Ohm modal (#864)

* Add OTC offer contract

* cleanup, add tests

* deploy scripts

* update post review

* natspec title

* review feedback

* testnet deploy

* Add OTC Ohm modal

* Use BigNumber for allowance

* Check max DAI available for OTC

* Add prod otcOffer address

---------

# Description
What does this PR solve?

Fixes # (issue)

# Checklist
- [ ] This PR is targeting the correct branch
